cmd/juju/ssh/debugcode.go

```diff
@@ -29,27 +37,36 @@ type debugCodeCommand struct {
 }
 
 const debugCodeDoc = `
-Interactively debug hooks and actions on a unit.
+The command launches a tmux session that will intercept matching events and/or 
```

Comment:
I can do that.  But just for the sake of explanation: I tried to be precise here, because technically event names are given as arguments here.
A hook is an [event handler](https://juju.is/docs/sdk/hook) and from a Charm developer's point of view there could be several event handlers (hooks) for one event.

I just discovered that the terms hook and event are also used interchangeably in the SDK documentation (e.g. 
[here](https://juju.is/docs/sdk/charm-environment-variables)), and the `JUJU_HOOK_NAME` environment variable also seems to contain the event name, so it may be normal in the Juju world, although I have to admit it did confuse me a bit.